lib/engine/step/auctioner.rb

```diff
@@ -77,6 +77,10 @@ def max_bid(_entity, _company)
         raise NotImplementedError
       end
 
+      def can_bid?(_entity, _company)
+        true
```

Comment:
Add `return false if max_bid(entity, company) < min_bid(company)` to maintain the old logic.

Also, make sure to test at least one other game to verify your changes didn't break the bidding UI.

Reply:
Hmm there was not old logic its a new function. But yes it makes sense to have that locig there.

Reply:
The logic was moved out of the UI in the refactor, but it was not placed here. That's what I'm referring too. We still need to maintain the same behavior.

Reply:
I really dont follow, the render_bid_input method is new to this PR. Therefore the logic in it can not be old.

Reply:
But i agree the default behaviour should be the logic of `max_bid(entity, company) < min_bid(company)`

Reply:
My apologies. I confused the file with ` assets/app/view/game/bid.rb`. Thanks for clarifying. 

Since 1822 is the only current game to access bid from the stock round, you can either remove `can_bid?` from the base auctioner or add in the basic logic outlined above. Whichever way you prefer.

Reply:
There are more games to use bid in the stockround, just not the way i do it. In method `render_pre_ipo` there is a line 

`children << h(Bid, entity: @current_entity, corporation: corporation) if @current_actions.include?('bid')`

Im guessing its from 1817.

Reply:
Since in the base auctioneer the methods `min_bid` and `max_bid` do a `raise NotImplementedError`. Shall i do the same for `can_bid?`